acceptance/tests/resource/service/service_enable_linux.rb

```diff
@@ -13,9 +13,6 @@
                 'ubuntu' => 'apache2',
 }
```

Comment:
we can confine tests based on packages?  how?
maybe you mean one test each for httpd and apache2 platforms?

if that's true maybe we should confine based on provider?